src/python/grpcio/grpc/_cython/_cygrpc/grpc_gevent.pyx.pxi

```diff
@@ -15,13 +15,17 @@
 
 from libc cimport string
 import errno
+import sys
 gevent_g = None
 gevent_socket = None
 gevent_hub = None
 gevent_event = None
 g_event = None
 g_pool = None
 
+cdef grpc_error* grpc_error_cancelled():
+  return <grpc_error*>4
```

Comment:
The ground truth is at "src/core/lib/iomgr/error.h". We should import it via `grpc.pxi`. It helps the code health.